CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 - 
-## [4.11.3] - 2022-11-02
+## [4.11.3] - 2022-11-17
```

Comment:
will this pass deployment, or why is the date changed?

Reply:
Date was wrong. This was released today.